pkg/api/types.go

```diff
@@ -1489,6 +1556,9 @@ type ServiceSpec struct {
 	// those endpoints.
 	Selector map[string]string `json:"selector"`
 
+	// DisruptionBudget defines the max disruption that can be caused to the collection of pods the Service matching
+	DisruptionBudget *DisruptionBudget `json:"disruptionBudget,omitempty"`
```

Comment:
DisruptionBudget is in Service and Job, which is like apples and oranges.

Additionally, this is ServiceSpec, but DisruptionBudget contains both spec and status.